src/enclave/virtual_enclave.h

```diff
@@ -2,27 +2,28 @@
 // Licensed under the Apache 2.0 License.
 #pragma once
 
+#ifndef CCHOST_SUPPORTS_VIRTUAL
```

Comment:
Renamed this file. It was `ccf_v.h` to indicate it was a redefinition of `ccf_u.h`/`ccf_t.h`, but actually that restriction was unnecessary. While doing this, I've removed a bunch of unused definitions from this file.